File: EGovernment_backend/statistics-service/domain/response.go
package domain

import (
	"go.mongodb.org/mongo-driver/bson/primitive"
)

type Response struct {
	ID         primitive.ObjectID `bson:"_id,omitempty" json:"id"`
	Text       string             `bson:"text" json:"text"`
	Attachment string             `bson:"attachment" json:"attachment"`
	Accepted   bool               `bson:"accepted" json:"accepted" `
	SendTo     User               `bson:"send_to" json:"send_to"`
	Date       primitive.DateTime `bson:"date" json:"date"`
}

File: EGovernment_frontend/src/app/models/statisics/response.ts
export class Response {
  id: string;
  text: string;
  attachment: string;
  accepted: boolean;
  send_to: string;
  date: string;
 
  constructor(id: string, text: string, attachment: string, accepted: boolean, send_to: string, date: string) {
    this.id = id;
    this.text = text;
    this.attachment = attachment;
    this.accepted = accepted;
    this.send_to = send_to;
    this.date = date;
  }
}